src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -0,0 +1,170 @@
+import React, { useCallback } from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconMail,
+  IconTrash,
+  Info,
+  Link,
+  Switch,
+  useTheme,
+  Split,
+  Tag,
+  textStyle,
+} from '@aragon/ui'
+
+import emailNotifcationIllustration from '../../../../src/assets/emailNotifications.svg'
+
+export default function NotificationsPreferences({
+  email,
+  notificationsDisabled,
+  onSwitchNotificationsStatus,
+  onLockSettings,
+  onDeleteEmail,
+  onUpdateEmail,
+}) {
+  const theme = useTheme()
+
+  return (
+    <React.Fragment>
+      <Split
+        primary={
+          <EmailNotificationBox
+            notificationsDisabled={notificationsDisabled}
+            onSwitchNotificationsStatus={onSwitchNotificationsStatus}
+          />
+        }
+        secondary={
+          <React.Fragment>
+            <Box heading="Email account">
+              <Tag
+                uppercase={false}
+                background={theme.infoSurface}
+                size="normal"
+                color={theme.content}
+                css={`
+                  ${textStyle('body2')};
+                  border-radius: 3px;
+                `}
+              >
+                {email}
+              </Tag>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+                wide
+                mode="strong"
+                onClick={onLockSettings}
+              >
+                Lock settings
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
+                `}
+                wide
+                onClick={onUpdateEmail}
+              >
+                <IconMail />
+                Update email
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
+                `}
+                wide
+                onClick={onDeleteEmail}
+              >
+                <IconTrash
+                  css={`
+                    color: ${theme.negative};
+                  `}
+                />
+                Delete your email
+              </Button>
+              <Info
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+              >
+                Learn more about our{' '}
+                <Link href="https://aragon.one/email-collection.md">
+                  Email collection policy
```

Comment:
```suggestion
                  email collection policy
```

I think elsewhere we don't capitalize "email".